pkg/argparse/args.go

```diff
@@ -0,0 +1,117 @@
+package argparse
+
+import (
+	"flag"
+	"os"
+	"strings"
+
+	"github.com/kitex-contrib/kitexcall/pkg/config"
+	"github.com/kitex-contrib/kitexcall/pkg/errors"
+	"github.com/kitex-contrib/kitexcall/pkg/log"
+)
+
+type Augment struct {
+	config.Config
+}
+
+type EndpointList []string
+
+func (e *EndpointList) String() string {
+	return strings.Join(*e, ",")
+}
+
+func (e *EndpointList) Set(value string) error {
+	*e = append(*e, value)
+	return nil
+}
+
+func (a *Augment) buildFlags() *flag.FlagSet {
+	f := flag.NewFlagSet(os.Args[0], flag.ContinueOnError)
+	f.StringVar(&a.Type, "type", "unknown", "Specify the type of IDL: 'thrift' or 'protobuf'.")
+	f.StringVar(&a.Type, "t", "unknown", "Specify the type of IDL: 'thrift' or 'protobuf'. (shorthand)")
+
+	f.StringVar(&a.IDLPath, "idl-path", "", "Specify the path of IDL file.")
+	f.StringVar(&a.IDLPath, "p", "", "Specify the path of IDL file. (shorthand)")
+
+	f.StringVar(&a.Service, "service", "", "Specify the service name.")
+	f.StringVar(&a.Service, "s", "", "Specify the service name. (shorthand)")
+
+	f.StringVar(&a.Method, "method", "", "Specify the method name.")
+	f.StringVar(&a.Method, "m", "", "Specify the method name. (shorthand)")
+
+	f.StringVar(&a.Data, "data", "", "Specify the data to be sent.")
```

Comment:
So it's the full json? Better allow input via file (or both).

Reply:
done